src/app/components/videochat/jh-video-chat.directive.js

```diff
@@ -122,7 +122,11 @@
           }
           // If current one is still speaking, there is no need to change
           if ( !(current && current.getSpeaking()) ) {
-            vm.highlight.current = FeedsService.speakingFeed() || current || FeedsService.findMain();
+            if (FeedsService.speakingFeed() && FeedsService.speakingFeed().getVideoSubscription()){
```

Comment:
You should not use `getVideoSubscription` here. That function only works for publisher feeds (i.e. yourself and your screen sharings). Use something like `someFeed.isEnabled("video")` instead. That function will always give you the right value, no matter if it's a publisher or subscriber feed.